package.json

```diff
@@ -7,7 +7,7 @@
     "start": "webpack -w & nodemon server",
     "start-server": "nodemon server",
     "build-client": "webpack",
-    "test": "mocha --compilers js:babel-register --watch-extensions js,jsx server/**/*.test.js"
```

Comment:
@aperez25 — careful, this will search through your `node_modules`. Even if there are no `spec` files in there, there are so many thousands of folders that this will slow down your test script.